src/migration/tasks/RepairTransactionPricedAtZeroTask.ts

```diff
@@ -0,0 +1,152 @@
+import { PricingSettings, PricingSettingsType } from '../../types/Setting';
+import Tenant, { TenantComponents } from '../../types/Tenant';
+import global, { ActionsResponse } from '../../types/GlobalType';
+
+import Constants from '../../utils/Constants';
+import ConsumptionStorage from '../../storage/mongodb/ConsumptionStorage';
+import Logging from '../../utils/Logging';
+import { ServerAction } from '../../types/Server';
+import SettingStorage from '../../storage/mongodb/SettingStorage';
+import TenantMigrationTask from '../TenantMigrationTask';
+import Transaction from '../../types/Transaction';
+import TransactionStorage from '../../storage/mongodb/TransactionStorage';
+import Utils from '../../utils/Utils';
+import chalk from 'chalk';
+
+const TASK_NAME = 'RepairTransactionPricedAtZeroTask';
```

Comment:
No constant in other classes, no constant in this one either. Or change all the other classes ;-)